pkg/nats/delay.go

```diff
@@ -0,0 +1,28 @@
+package nats
+
+import (
+	"time"
+)
+
+// StopTime if this duration was returned, event will be term`ed
+const StopTime = time.Duration(-1)
```

Comment:
Do you think that using "Term" in name of this const may make it more explicit?

It would be also nice to add some context, how it could be used. I guess that it can be returned from `WaitTime` when we want to term message?

Reply:
This is one that came in a jet stream PR and I haven't gotten my head around it fully yet - I think TermSignal makes sense?  Added a max retry delay and some simple tests to illustrate usage too: https://github.com/ThreeDotsLabs/watermill-nats/pull/8/commits/1e63667efe7fdac2c1d7df9c774c5b98bf735dc6

Reply:
nice 👍